test/runtests.jl

```diff
@@ -14,6 +14,7 @@ const is_APPVEYOR = Sys.iswindows() && haskey(ENV,"APPVEYOR")
     @time @safetestset "Static Array Tests" begin include("static_array_tests.jl") end
     @time @safetestset "Noise Type Tests" begin include("noise_type_test.jl") end
     @time @safetestset "Mass matrix tests" begin include("mass_matrix_tests.jl") end
+    #@time @safetestset "Sparse Jacobian tests" begin include("sparsediff_tests.jl") end
```

Comment:
why did this get commented out?

Reply:
No, it wasn't there at all.  I decided to add it as a commented out line so it is not forgotten.  I didn't check whether those tests actually work on master.  They do fail on this PR though.

Reply:
I now ran that file on master as well and it gives the same failure.  On L49 the error is too big:
```
julia> maximum(abs.(sol_sp.u[end].-sol.u[end]))
0.0879739350640969
```
and on the next line the lengths are different
```
julia> length(sol_sp.t), length(sol.t)
(28, 34)
```
Similarly for the other two tests.

Reply:
Oh I see... that test is bad. It's using a different Brownian process between the two, so of course the solution is stochastic haha. That needs some work.

Reply:
I thought there should be some random seed needing to be set.  So this is ready to be merged, I think.

Reply:
add seed=1 to `solve` and see?

If it doesn't work, I can take it from here, but I would like to know the result.

Reply:
No, that doesn't help, at all.  On master I still get the identical numbers I reported above.